var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -75,16 +90,16 @@ class Julia(MakefilePackage):
         depends_on('openlibm@0.7.0:0.7', when='+openlibm')
 
     # Patches for llvm
-    depends_on('llvm', patches='llvm7-symver-jlprefix.patch')
+    depends_on('llvm', patches='llvm7-symver-jlprefix.patch', when='@:1.7')
     depends_on('llvm', when='^llvm@11.0.1', patches=patch(
         'https://raw.githubusercontent.com/spack/patches/0b543955683a903d711a3e95ff29a4ce3951ca13/julia/llvm-11.0.1-julia-1.6.patch',
         sha256='8866ee0595272b826b72d173301a2e625855e80680a84af837f1ed6db4657f42'))
     depends_on('llvm', when='^llvm@12.0.1', patches=patch(
         'https://github.com/JuliaLang/llvm-project/compare/fed41342a82f5a3a9201819a82bf7a48313e296b...980d2f60a8524c5546397db9e8bbb7d6ea56c1b7.patch',
         sha256='10cb42f80c2eaad3e9c87cb818b6676f1be26737bdf972c77392d71707386aa4'))
-    depends_on('llvm', when='^llvm@13.0.0', patches=patch(
-        'https://github.com/JuliaLang/llvm-project/compare/d7b669b3a30345cfcdb2fde2af6f48aa4b94845d...6ced34d2b63487a88184c3c468ceda166d10abba.patch',
-        sha256='92f022176ab85ded517a9b7aa04df47e19a5def88f291e0c31100128823166c1'))
+    depends_on('llvm', when='^llvm@13.0.1', patches=patch(
+        'https://github.com/JuliaLang/llvm-project/compare/75e33f71c2dae584b13a7d1186ae0a038ba98838...5393efbd8a4c7555b9f9fdf185c486c6b05f0c19.patch',
```

Comment:
Needs to be https://github.com/JuliaLang/llvm-project/releases/tag/julia-13.0.1-1 which I am in process of backporting.

It includes a fix for Summit.